app/controllers/contents_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+# Controller for a Work contents (files)
+class ContentsController < ApplicationController
+  # Skipping authorization for now
+  skip_verify_authorized only: %i[show edit update]
+  before_action :set_content, only: %i[edit update show]
+
+  def show
+    @content_files = @content.content_files
+  end
+
+  def edit; end
+
+  def update
+    if params.dig(:content, :files).present?
+      update_files
+      return respond_to do |format|
+        format.turbo_stream
+      end
+    end
+
+    redirect_to edit_content_path(@content)
```

Comment:
Also uncovered. Worth testing?

Reply:
I actually don't think this is necessary, so removing.